pkger/service_test.go

```diff
@@ -1576,7 +1576,7 @@ func TestService(t *testing.T) {
 					EditMode: "advanced",
 				}
 				// TODO: remove this when issue that forced the builder tag to be here to render in UI.
-				q.BuilderConfig.Tags = append(q.BuilderConfig.Tags, influxdb.NewBuilderTag("_measurement"))
+				q.BuilderConfig.Tags = append(q.BuilderConfig.Tags, influxdb.NewBuilderTag("_measurement", ""))
```

Comment:
can we axe this now?

Reply:
@hoorayimhelping are we good to go on the builder tags issue we spke of a few days back?

Reply:
@jsteenb2 i don't think so. we might be looking at it now, but i'm not sure

Reply:
no sweat, thanks for the heads up 👍